pkg/config/config.go

```diff
@@ -111,6 +111,11 @@ type OIDCIssuer struct {
 	// Optional, the challenge claim expected for the issuer
 	// Set if using a custom issuer
 	ChallengeClaim string `json:"ChallengeClaim,omitempty" yaml:"challenge-claim,omitempty"`
+	// Optional, the description for the issuer
```

Comment:
We want to omit this fields when querying the API, so we need to define another struct without these fields and wire that up as well.

Reply:
hmm, is the issuer retuned from the API this one? https://github.com/sigstore/fulcio/blob/b879601e1aaf48de01f2997ba7be339377b7b93e/pkg/config/config.go#L229. because it seems that this one comes from a proto, that omits this information. I also tested it by calling the api locally and the information was omitted.

Reply:
<img width="1296" alt="image" src="https://github.com/user-attachments/assets/6cca2140-7afa-4c43-b967-4dc3357a732b">

Example of api's return

Reply:
Ah yep, you're right! All good then, thanks.